packages/logging_cloudwatch/aws_logging_cloudwatch/lib/src/queued_item_store/queued_item_store.dart

```diff
@@ -38,6 +38,9 @@ class QueuedItem {
     required this.timestamp,
   });
 
+  /// The size of the item, in bytes, when stored in the database.
+  int get byteSize => value.length + timestamp.length + 8; // 8 bytes for id
```

Comment:
instead of calculating the size each time on the getter, isn't it better to use a private prop and return it from the getter? 

```suggestion
  const QueuedItem({
    required this.id,
    required this.value,
    required this.timestamp,
  }) : _byteSize = value.length + timestamp.length + 8;
  final int _byteSize;
  int get byteSize => _byteSize;
```